server/command.go

```diff
@@ -125,6 +125,12 @@ func (c *Conn) dispatch(data []byte) interface{} {
 		} else {
 			return r
 		}
+	case COM_SET_OPTION:
+		if err := c.h.HandleOtherCommand(cmd, data); err != nil {
```

Comment:
Currently this is always `EmptyHandler`, so always returns an error 🤔 . Is this your expectation?

Reply:
Well in my server implementation I implemented a Handler that does handle these commands. And `EmptyHandler` would've returned an error for this anyway.

Or perhaps I don't understand what you mean?